src/audio/codec_adapter/codec/cadence.c

```diff
@@ -20,7 +20,27 @@
 static struct cadence_api cadence_api_table[] = {
 	{
 		.id = 0x01,
-		.api = cadence_api_function
+		.api = xa_aac_dec,
+	},
+	{
+		.id = 0x02,
```

Comment:
I assume all these are in fact different cadence codecs performing different processing?

Reply:
yes.

Reply:
@mrajwa one more thing. I think we can still keep cadence_api_function symbol as a wildcard.

Reply:
@dbaluta yes good idea, please don't change the 0x01 ID and keep is as a wildcard.